string state
bool ball_is_handled
int32 ball_dis
# geometry_msgs/Twist position